integration/src/test/java/com/aws/greengrass/mqtt/moquette/ClientDeviceAuthorizerTest.java

```diff
@@ -232,4 +250,53 @@ authorizer.new ConnectionTerminationListener()
         verify(mockClientDevicesAuthService).closeClientDeviceAuthSession(DEFAULT_SESSION);
         assertThat(authorizer.getSessionForClient(DEFAULT_CLIENT, DEFAULT_PEER_CERT), nullValue());
     }
+
+    @Test
+    void GIVEN_authorizedClientAndInvalidSession_WHEN_checkValid_THEN_returnsTrue(ExtensionContext context) throws
+        AuthenticationException, AuthorizationException {
+        ignoreExceptionOfType(context, InvalidSessionException.class);
```

Comment:
if we're logging too much for invalid sessions, we should re-evaluate how much to log and at what level.

What does the log look like right now when there's an invalid session? For example, we could consider logging only the message and not the full stacktrace if the stacktrace won't ever be helpful to us.

Reply:
Currently when there is an invalid session the log looks as follows:
`[WARN] (main) com.aws.greengrass.mqtt.moquette.ClientDeviceAuthorizer: Session ID is invalid. {sessionId=SESSION_ID}
com.aws.greengrass.clientdevices.auth.exception.InvalidSessionException: Invalid session ID (SESSION_ID)
at com.aws.greengrass.clientdevices.auth.api.ClientDevicesAuthServiceApi.authorizeClientDeviceAction(ClientDevicesAuthServiceApi.java:89)
at com.aws.greengrass.mqtt.moquette.ClientDeviceAuthorizer.canDevicePerform(ClientDeviceAuthorizer.java:157)
at com.aws.greengrass.mqtt.moquette.ClientDeviceAuthorizer.checkValid(ClientDeviceAuthorizer.java:68)
at com.aws.greengrass.mqtt.moquette.ClientDeviceAuthorizerTest.GIVEN_authorizedClientAndInvalidSession_WHEN_checkValid_THEN_returnsTrue(ClientDeviceAuthorizerTest.java:263)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at java.util.ArrayList.forEach(ArrayList.java:1259)`

with ~60 internal lines

Reply:
Right, that's a lot of useless info. Let's just log the exception type and message instead of the entire exception.

example: `log.error("{}: {}", e.getClass().getSimpleName(), e.getMessage())`